pipelines/skylab/multiome/Multiome.wdl

```diff
@@ -100,6 +107,6 @@ workflow Multiome {
         File cell_metrics_gex = Optimus.cell_metrics
         File gene_metrics_gex = Optimus.gene_metrics
         File? cell_calls_gex = Optimus.cell_calls
-        File h5ad_output_file_gex = Optimus.h5ad_output_file
+        File h5ad_output_file_gex = JoinBarcodes.gex_h5ad_file
```

Comment:
Does this change the naming of the output file?

Reply:
No, the actual file names stay the same